ios/Podfile

```diff
@@ -1,11 +1,13 @@
 require_relative '../node_modules/react-native/scripts/react_native_pods'
 require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
 
-platform :ios, '12.0'
+platform :ios, '12.4'
```

Comment:
Is this a new RN requirement?

Reply:
Yes, starting 0.69, iOS 12.4 is minimum supported version;
https://reactnative.dev/blog/2022/06/21/version-069#highlights-of-069